README.md

```diff
@@ -0,0 +1,44 @@
+![Aragon Protocol](./docs/aragon-protocol.png)
+
+<img align="right" src="https://github.com/aragon/protocol/workflows/CI/badge.svg">
+  <a href="https://github.com/aragon/protocol/actions"/>
+</img>
+
+## Project
+
+#### 👩‍️ [Become an Aragon Protocol guardian](https://ant.aragon.org)
+Aragon Protocol is now live on Ethereum mainnet. You can become a guardian by staking ANT.
+
+#### ⚖ [Check out the Aragon Protocol Dashboard](https://protocol.aragon.org)
+The Aragon Protocol Dashboard is the central app where all dispute-related information is available for guardians.
+
+#### 📚 [Read the User Guide](https://help.aragon.org/category/47-aragoncourt) 
```

Comment:
Can probably leave this out for now and add an issue to re-add it later?

Reply:
Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20